comictaggerlib/seriesselectionwindow.py

```diff
@@ -465,24 +465,24 @@ def categorize(result: ComicSeries) -> int:
             self.twList.setItem(row, 0, item)
 
             item_text = str(record.start_year)
-            item = QtWidgets.QTableWidgetItem(item_text)
+            item = QtWidgets.QTableWidgetItem(record.start_year)
```

Comment:
This sets the item [type] and should not be used instead pad to four characters: `f"{record.start_year:04}"`

[type]: https://doc.qt.io/qt-6/qtablewidgetitem.html#QTableWidgetItem